ui/js/directives/treeherder/clonejobs.js

```diff
@@ -360,17 +371,46 @@ treeherder.directive('thCloneJobs', [
                     countInfo.value = countInfo.count;
                     countInfo.title = countInfo.count + " " + countInfo.countText + " jobs in group";
                     countInfo.btnClass = countInfo.btnClass + "-count";
```

Comment:
Could we not perform this logic inside the interpolator, instead of assigning to a temporary? Seems to me like that would be more clear.